<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindows</class>
 <widget class="QMainWindow" name="MainWindows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sensores</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>331</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Graf1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>sensor 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Graf2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>sensor 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Graf3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>sensor 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Graf4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>sensor 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Salir">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
background-color: rgb(170, 170, 170);</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>imagenes/casa.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Graf">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>sensores</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>553</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Salir</sender>
   <signal>clicked()</signal>
   <receiver>MainWindows</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
